core-bundle/contao/classes/BackendUser.php

```diff
@@ -132,6 +132,9 @@ public function __get($strKey)
 
 			case 'fop':
 				return \is_array($this->arrData['fop']) ? $this->arrData['fop'] : ($this->arrData['fop'] ? array($this->arrData['fop']) : false);
+
+			case 'alexf':
```

Comment:
Should we add all other array-like properties (see class annotations)?

Reply:
@leofeyer wdyt? We already do it for some of the properties, but not all.

Reply:
If we are certain that it does not cause more issue, then yes.

Reply:
As a bugfix I’d only handle the `alexf` case. The other properties can be done in 5.5 IMO.

Reply:
I am not sure. Furthermore, for some of these properties we return `false` instead of an empty array. So it is inconsistent right now, not matter what. And I don't think we should change that as a bugfix - I think we should just fix the original issue.